quarto/config/packages/dev/web_profiler.yaml

```diff
@@ -1,5 +1,5 @@
 web_profiler:
-    toolbar: true
+    toolbar: false
```

Comment:
With `Symfony`, you have to set a production environment for ... production.
https://symfony.com/doc/4.0/configuration.html#environments-the-other-config-files